worlds/sa2b/docs/setup_pt-br.md

```diff
@@ -0,0 +1,140 @@
+
+Nota: Tenha certeza de executar Sonic Adventure 2 diretamente da Steam no Linux, ao invés de executar usando o botão `Salvar & Jogar` do SA Mod Manager.
+
+## Entrando em um jogo MultiWorld
+
+1. Antes de abrir o jogo, execute o `SAModManager.exe`, selecione o mod SA2B_Archipelago, e aperte o botão `Configurar Mod`.
+
+2. Para o campo `Server IP` em  `AP Settings`, coloque o endereço do servidor, como por exemplo archipelago.gg:38281, o host do servidor deve conseguir identificar isso.
+
+3. Para o campo `PlayerName` em `AP Settings`, insira o "nome" que inseriu no yaml, ou no website config.
+
+4. Para o campo `Password` em `AP Settings`, insira a senha do servidor se uma existir, caso o contrário deixe em branco.
+
+5. Clique no botão `Salvar` e aperte `Salvar & Jogar` para abrir o jogo. No Linux, abra Sonic Adventure 2 diretamente da Steam ao invés de apertar `Salvar & Jogar`.
+
+6. Crie um novo save para se conectar a um jogo MultiWorld. Uma mensagem dizendo "Connected to Archipelago" irá aparecer se você se conectar com sucesso. Se você fechar o jogo enquanto conectado, você pode se reconectar ao jogo MultiWorld selecionando o mesmo save slot.
+
+## Opções Adicionais
+
+Algumas opções adicionais relacionadas às mensagens do Archipelago dentro do jogo podem ser ajustadas no SAModManager se você selecionar `Configurar Mod` no mod SA2B_Archipelago. Essas opções estarão na aba `General Settings`.
+	
+- Message Display Count: Esse é o número máximo de mensagens do Archipelago que podem ser exibidas na tela a qualquer ponto.
+- Message Display Duration: Isso dita por quanto tempo as mensagens do Archipelago são exibidas na tela (em segundos).
+- Message Font Size: Isso é o tamanho da fonte usada para exibir as mensagens do Archipelago.
+
+Se você deseja usar a opção `SADX Music` do Randomizer, você deve possuir uma cópia de `Sonic Adventure DX` na Steam e seguir os seguintes passos:
+
+1. Ache a pasta no seu PC onde `Sonic Adventure DX` está instalado.
+
+2. Abra a pasta `SoundData` na pasta de instalação de `Sonic Adventure DX`, e copie a pasta `bgm`.
+
+3. Dentro da pasta do mod `SA2B_Archipelago`, cole a pasta `bgm` na pasta `ADX` que existe dentro da pasta `gd_PC`.
+
+## Solução de problemas
+
+- "The following mods didn't load correctly: SA2B_Archipelago: DLL error - The specified module could not be found."
+	- Tenha certeza que `APCpp.dll` está na mesma pasta que `sonic2app.exe`. (Veja Passo a Passo de Instalação passo 6)
+	
+- "sonic2app.exe - Entry Point Not Found"
+	- Tenha certeza que `APCpp.dll` está atualizado para a última versão. Siga Passo a Passo de Instalação passo 6 para atualizar o dll.
+
+- Jogo rodando muito rápido (Que nem o Sonic).
+	- Limite o framerate usando o mod manager:
+		1. Execute `SAModManager.exe`.
+		2. Selecione a aba `Definições do Jogo`, e então selecione a subaba `Correções`.
+		3. Marque a caixinha do `Lock Framerate` na seção Correções.
```

Comment:
didn't use the program, but if its translated or not i believe that everthing in ` ` should be the same language. If not translated, it should also not be translated here

Reply:
There isn't a translation in for the `Lock Framerate` option _specifically_, yeah
![image](https://github.com/user-attachments/assets/0c2f9b3d-8f8f-4cee-893f-86cd28a642b0)